WordPress/gutenberg
# [Mobile] - Global styles - Add support to render font sizes and line height

`Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/3924

## Description
This PR is a follow-up of https://github.com/WordPress/gutenberg/pull/33605 and adds the functionality to render custom font sizes and line height in the editor. 

This is under the `DEV` flag.

## How has this been tested?

**Notes to test in a dev build**:
- For iOS make sure to clear the build folder and on Android to build Gutenberg Mobile locally.
- To test the line-height feature you need a theme that has this enabled, like the Quadrat theme.

### Test case 1 (font size)

- Open the app
- Add a Paragraph block with a few lines of text
- Open the block settings
- **Expect** to see the Font size selector
- Select a font size
- **Expect** the value to change accordingly

### Test case 2 (line-height)

- Open the app
- Add a Paragraph block with a few lines of text
- Open the block settings
- **Expect** to see the line-height selector
- Change the line-height value
- **Expect** the value to change accordingly in the block

## Screenshots 

Test case 1 iOS|Test case 1 Android
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/132348764-d5f5e94a-240e-433d-99c6-6a67faeccb5f.gif" width="200" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/132349118-72e8810c-f323-4f73-8a49-d9484762d909.gif" width="200" /></kbd>

Test case 2 iOS|Test case 2 Android
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/132354976-e58ba682-1b26-4b25-928c-00c90b8e91c8.gif" width="200" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/132354981-d5ed65fa-4d4c-425f-8d9a-f40f9f8ae2b5.gif" width="200" /></kbd>

## Types of changes





## Checklist:
